code/src/components/NewThought.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+
+// useState
+
+export const NewThought = () => {
+    const [thoughts, setThoughts] = useState ([])
+    const [newThought, setNewThought] = useState('')
+
+
+//event
+
+    const onFormSubmit = (event) => {
+        event.preventDefault()
+
+        const options = {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({message: newThought})
+        }
+
+//fetch url
+
+        fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts', options)
+            .then((res) => res.json())
+            .then((data) => setThoughts([data, thoughts])) 
+    }
+
+    return (
+        <>
+            <form className="form" onSubmit={onFormSubmit}>
+                <label>What's making you happy right now?</label>
+                <textarea 
+                    className="input-field"
+                    rows="4"
```

Comment:
This is a great way of making the input field bigger! I forgot to add that one myself